src/kubernetes_configmap_jwt_scan/kubernetes_configmap_jwt_scan/scan.py

```diff
@@ -14,7 +14,7 @@
 from lib.kctl import Kctl
 
 WARN_THRESHOLD = 30 # warn if expiring within 30 days
-JWT_REGEX = re.compile('^(?:[\w-]*\.){2}[\w-]*$')
+JWT_REGEX = re.compile(r'^(?:[\w-]*\.){2}[\w-]*$')
```

Comment:
Added the `r` here as I was getting:

> DeprecationWarning: invalid escape sequence \w
    JWT_REGEX = re.compile('^(?:[\w-]*\.){2}[\w-]*$')

[Solution source](https://stackoverflow.com/a/50504635)